apps/prairielearn/elements/pl-checkbox/pl-checkbox.mustache

```diff
@@ -1,6 +1,6 @@
 {{#question}}
 {{#inline}}<span class="d-inline-block">{{/inline}}
-{{^inline}}<div class="d-block">{{/inline}}
+{{^inline}}<div class="d-block" aria-label="checkbox options">{{/inline}}
```

Comment:
As discussed on Zoom, consider using a [`<fieldset>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) with a `<legend>` (possibly hidden from view and only visible to screenreaders). It'd be great to know how they compare.

Reply:
FWIW, Canvas uses `fieldset`/`legend` for both multiple choice and checkbox question types, and the (screenreader-only) label is "Group of answer choices". We can probably do better with the label, but overall I think `fieldset` is probably the way to go.

Reply:
Updated to `<fieldset>` and `<legend>` here and in the multiple choice element in [b2ddc80](https://github.com/PrairieLearn/PrairieLearn/pull/10514/commits/b2ddc80c9af2e30a4fa6ea94027150ce2ac8b3ba).